test/clearValueTest.js

```diff
@@ -0,0 +1,17 @@
+describe('clearValue Demo', function() {
+  before(browser => {
+    browser
+      .navigateTo('https://sandbox.mabl.com/mailbox');
+  });
+  
+  it('shows issue with clearValue command', async function(browser) {
+    browser
+      .sendKeys('input[name=to]', 'hello')
+      .clearValue('input[name=to]')
+      .sendKeys('input[name=to]', 'hi')
+      .pause(3000);
+  });
+  
+  after(browser => browser.end());
+  
+});
```

Comment:
What is this testing? I think we should add a unit test for the change

Reply:
Surely thanks again for guiding me i will surely add unit test and remove this.